src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/FallbackProjectManager.cs

```diff
@@ -81,7 +87,7 @@ private void AddFallbackProject(ProjectId projectId, string filePath)
             return;
         }
 
-        var rootNamespace = project.DefaultNamespace ?? "ASP";
+        var rootNamespace = project.DefaultNamespace;
```

Comment:
This was copying behaviour of the source generator, which is the same as what I did in the VS Code work, but its not necessary as everything this ends up going through supports null. Feels better to let the compiler pick a default in case it has better smarts about doing so, or gets some in future.